lxd/api_1.0.go

```diff
@@ -617,6 +619,25 @@ func api10Patch(d *Daemon, r *http.Request) response.Response {
 	return doAPI10Update(d, r, req, true)
 }
 
+func getProjectNameFromStorageConfig(config string) string {
```

Comment:
this function would be a good candidate for a unit test as its self contained. Not a blocker though just an observation.